Gemfile

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+ruby '2.4.0'
```

Comment:
I also had to change `FROM ruby:2.4.1` in the Dockerfile to `FROM ruby:2.4.0` to avoid `Your Ruby version is 2.4.1, but your Gemfile specified 2.4.0`